sign_test.go

```diff
@@ -19,10 +18,10 @@ import (
 func TestSign(t *testing.T) {
 	content := []byte("Hello World")
 	sigalgs := []x509.SignatureAlgorithm{
-		x509.SHA1WithRSA,
+		// x509.SHA1WithRSA,
```

Comment:
I noticed tests that used the now commented out algorithms would fail with `insecure algorithm SHA1-RSA (temporarily override with GODEBUG=x509sha1=1)`. We can either remove these from the tests or `GODEBUG=x509sha1=1` to the `test` Makefile target.